lib/browser.ts

```diff
@@ -138,9 +139,9 @@ export class ProtractorBrowser extends Webdriver {
    * The wrapped webdriver instance. Use this to interact with pages that do
    * not contain Angular (such as a log-in screen).
    *
-   * @type {webdriver.WebDriver}
+   * @type {ExtendedWebDriver}
    */
-  driver: WebDriver;
+  driver: ExtendedWebDriver;
```

Comment:
Looks like this is taken care of. This does not appear to have typing implications.